charts/nginx-ingress/README.md

```diff
@@ -100,15 +100,15 @@ CRDs](#upgrading-the-crds).
 To upgrade the release `my-release`:
 
 ```console
-helm upgrade my-release oci://ghcr.io/nginxinc/charts/nginx-ingress --version 1.0.1
+helm upgrade my-release -n nginx-ingress oci://ghcr.io/nginxinc/charts/nginx-ingress --version 1.0.1
```

Comment:
according to old command the installation was in default ns and now user is running upgrade in a different ns? I understand that these command as a guide but most users usually straight-up copy-paste and upgrade will fail due to ownership of clusterroles as well.

Reply:
How do you propose we change it?